exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
+  ["",                         ""],
+  // single C nucleotide strand
+  ["C",                       "G"],
+  // single G nucleotide strand
+  ["G",                       "C"],
+  // single A nucleotide strand
+  ["A",                       "U"],
+  // single T nucleotide strand
+  ["T",                       "A"],
+  // longer strand
+  ["ACGTGGTCTTAA", "UGCACCAGAAUU"],
+  // strand with invalid nucleotides
+  ["XCGFGGTDTTAA",        Error()],
```

Comment:
That's true; I always figured the purpose of providing the dummy class was more of a restriction as part of the solution for the problem.

I can adjust the tests to more similarly match Hamming's¸ if you'd prefer.